src/System.Diagnostics.Debug/src/System.Diagnostics.Debug.csproj

```diff
@@ -90,6 +94,11 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
+    <TargetingPackReference Include="mscorlib" Condition="'$(TargetGroup)' == 'net46'" />
```

Comment:
Should these conditions be on the ItemGroup to help separate and reduce redundancy.